README.md

```diff
@@ -19,108 +19,40 @@ The Buildah package provides a command line tool which can be used to
 
 **[Changelog](CHANGELOG.md)**
 
-**Installation notes**
-
-Prior to installing Buildah, install the following packages on your linux distro:
-* make
-* golang (Requires version 1.8.1 or higher.)
-* bats
-* btrfs-progs-devel
-* bzip2
-* device-mapper-devel
-* git
-* go-md2man
-* gpgme-devel
-* glib2-devel
-* libassuan-devel
-* ostree-devel
-* runc (Requires version 1.0 RC4 or higher.)
-* skopeo-containers
-
-In Fedora, you can use this command:
+**[Installation notes](install.md)**
 
-```
- dnf -y install \
-    make \
-    golang \
-    bats \
-    btrfs-progs-devel \
-    device-mapper-devel \
-    glib2-devel \
-    gpgme-devel \
-    libassuan-devel \
-    ostree-devel \
-    git \
-    bzip2 \
-    go-md2man \
-    runc \
-    skopeo-containers
-```
+Buildah uses `runc` to run commands when `buildah run` is used, or when `buildah build-using-dockerfile`
+encounters a `RUN` instruction, so you'll also need to build and install a compatible version of
+[runc](https://github.com/opencontainers/runc) for Buildah to call for those cases.
 
-Then to install Buildah on Fedora follow the steps in this example:
+## Example 
 
+From (`./examples/lighttpd.sh`)[examples/lighttpd.sh]:
```

Comment:
This isn't showing as a link in the view page.  I think it needs to be:

From [examples/lighttpd.sh](`./examples/lighttpd.sh`):

Reply:
and md just resolved that in my comment.  You need to have the stuff in the brackets before the stuff in the parens.